modules/tinymce/src/core/main/ts/api/html/DomParser.ts

```diff
@@ -573,6 +573,8 @@ const DomParser = (settings: DomParserSettings = {}, schema = Schema()): DomPars
     // Parse and sanitize the content
     const element = parseAndSanitizeWithContext(html, rootName, args.format);
 
+    TransparentElements.update(schema, element, Type.isUndefined(args.context));
```

Comment:
This doesn't look right. A context does not specify if it's for the root block, it should be using `isRootContent` instead from memory.

Reply:
Got rid of this in root thing. Transparents are now block like if they contain blocks. That is the only rule.